charts/rmd/templates/deployment-rmd.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "rmd.fullname" . }}
+  labels:
+    {{- include "rmd.labels" . | nindent 4 }}
+    app.kubernetes.io/component: rmd
+spec:
+  replicas: {{ .Values.rmd.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "rmd.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "rmd.labels" . | nindent 8 }}
+        app.kubernetes.io/component: rmd
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.rmd.image.repository }}:{{ default .Chart.AppVersion .Values.rmd.image.tag }}"
+          env:
+            - name: DJANGO_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "rmd.postgresql.fullname" . }}
+                  key: postgresql-password
```

Comment:
Does this still work if we set `postgresql.enabled=false`?

This occurs twice in the code ad you might want to split it into a helper function.